redbot/core/core_commands.py

```diff
@@ -326,9 +326,10 @@ class Core(commands.Cog, CoreLogic):
         """Shows Red's uptime"""
         since = ctx.bot.uptime.strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
Should be `bot._uptime` now

Reply:
No, this one was left available as a property which only errors if you try to assign to it.